sidebars.js

```diff
@@ -193,7 +193,10 @@ module.exports = {
       {
         type: "category",
         label: "Installation Guide (Node)",
-        items: ["operate-a-stake-pool/node-installation-process"],
+        items: [
```

Comment:
I wonder if this should be in deployment scenarios (as it would not be applicable to containerised solutions)

Reply:
I think containerised solutions are a kind of Deployment Scenarios. The guide is indeed more about hardening the Host, or a "traditional" server setup which is, i believe, inline with the "Node Installation process" guide

Reply:
I moved the Hardening Server section to "Basics"